### ISSUE TITLE
Crash when load lottie by url with extension ".json"

### ISSUE THREAD

**Issue by chihung93** [Aghajari/AXrLottie#1]
My Code: 
link url = https://files-5.gapo.vn/sticker/origin/234f279b-1342-4b8a-8673-5fc11e3bfbba.json
```
binding.imgSticker.lottieDrawable = AXrLottieDrawable.fromURL(stickerUrl)
                .setSize(size,size)
                .setAutoRepeat(true)
                .setAutoStart(true)
                .build()
```

My error console: 
`
java.net.MalformedURLException: no protocol: lottie_cache_httpsfiles5gapovnstickerorigin6525deecc18545a58adea943441c0bd9json`

Thanks.

**Comment by chihung93** [comment]
I have tried your example but not work
```

public class SimpleActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_simple);

        final AXrLottieImageView lottieView = findViewById(R.id.lottie_view);

       lottieView.setLottieDrawable(
               AXrLottieDrawable.fromURL("https://files-5.gapo.vn/sticker/origin/234f279b-1342-4b8a-8673-5fc11e3bfbba.json").setSize(140,140).build()
       );
       lottieView.playAnimation();

    }
}
```

**Comment by chihung93** [comment]
I think the problem is ` this.url = "lottie_cache_" + url.replaceAll("\\W+", "");`

**Comment by Aghajari** [comment]
Hi @chihung93 ,
You're right,
i have released v1.0.3 and fixed this issue.

> ```java
> implementation 'com.aghajari.rlottie:AXrLottie:1.0.3'
> ```

Your code works fine now :

<img width=200 src="https://user-images.githubusercontent.com/30867537/104184938-7ea3fa00-5429-11eb-91f6-3170cf748d77.jpg" title="Screen">

**Comment by chihung93** [comment]
@Aghajari  I think the lib is getting the wrong cache from the URL.
![2021-01-12 09 39 08](https://user-images.githubusercontent.com/5199109/104262396-1c470a00-54ba-11eb-93ab-0541f2c0036f.jpg)

**Comment by Aghajari** [comment]
Fixed on v1.0.4
